README.md

```diff
@@ -5,11 +5,12 @@ This extension allows ZenCart to use the Heartland Payment Systems Gateway. All
 ## Installation
 
 To install the development version, download and extract the files in this repo to the root of your ZenCart root folder. Activate and configure the plugin.
+Open the composer from the root directory and run the command "composer require globalpayments/php-sdk" 
```

Comment:
We need to ship a copy of the SDK within the plugin itself. As far as I know, typical ZenCart integrations do not leverage composer, so we cannot depend on the integrator having this available.

Reply:
updated